src/view/configuration/configuration.jsx

```diff
@@ -527,8 +543,50 @@ const Configuration = () => {
                             </div>
                           </div> */}
 
-                          <h3>Context</h3>
+                          <h3>Data Collection</h3>
 
+                          <div className="u-gapTop">
+                            <InfoTipLayout tip="Indicates whether data associated with clicks on navigational links, download links, or personalized content should be automatically collected or not.">
+                              <WrappedField
+                                name={`instances.${index}.clickCollectionEnabled`}
+                                component={Checkbox}
+                                label="Enable click data collection"
+                              />
+                            </InfoTipLayout>
+                          </div>
+                          {values.instances[index].clickCollectionEnabled ? (
+                            <div>
+                              <InfoTipLayout tip="Regular expression that qualifies a link URL as a download link.">
+                                <FieldLabel
+                                  labelFor="downloadLinkQualifier"
+                                  label="Download Link Qualifier"
+                                />
+                              </InfoTipLayout>
+                              <div>
```

Comment:
Let's add the class `FieldSubset` on this container, so that the disclosed download link qualifier field is inset a bit. This is what we did for the context checkboxes and was an agreed upon pattern with Laurel from UX.